### PR TITLE
Add case studies slider component

### PR DESCRIPTION
**Pull request by arzafran**
# Changes done
- add new component in order to display case studies

### CODE REVIEW

File: www/components/case-studies/slider.js [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+  }
+];
+
+const itemDuration = 3300;
+
+export default function CaseStudiesSlider() {
+  const [count, setCount] = useState(0);
+
+  if (items.length > 1) {
+    setTimeout(() => {
```
Comment (lfades) [inline www/components/case-studies/slider.js]: Can you move this to `useEffect` 🙏

Comment (arzafran) [reply www/components/case-studies/slider.js]: sure!

**Comment by lfades** [comment]
@arzafran Change the current `useEffect` it to this:

```jsx
useEffect(() => {
  if (items.length > 1) {
    const timer = setTimeout(() => {
      if (count < items.length - 1) {
        setCount(count + 1);
      } else {
        setCount(0);
      }
    }, itemDuration);

    return () => {
      clearTimeout(timer)
    }
  }
}, [count]);
```

It will clear the timeout if the component is unmounted, or when `count` changes.

File: www/components/case-studies/slider.js [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+          25%,
+          75% {
+            opacity: 1;
+          }
+          100% {
+            opacity: 0;
+          }
+        }
+
+        @media (max-width: 768px) {
```
Comment (lfades) [inline www/components/case-studies/slider.js]: Can this be lower ? when setting the device toolbar to `iPad` the case study switches to take the entire space, but the rest of the page remains the same, we should probably use the breakpoint used by other parts of the page.

Comment (arzafran) [reply www/components/case-studies/slider.js]: sure!

**Pull request merged**